### DISCUSSION TITLE
Default tie curvature

### DISCUSSION THREAD

**Discussion by infojunkie** [w3c-cg/musicxml#452] [Q&A] [answered]
As per https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/issues/1141, the `tied` element takes an optional `placement` attribute of `above` or `below`. The question is: if this attribute is not specified, what is the software expected to do? As per the discussion in this ticket, always assuming a default value does not produce good results. Is there a known rule to decide which tie placement to choose?

**Comment by mdgood** [comment, marked as answer]
Tie placement can get pretty complicated. _Beyond Bars_ devotes 12 pages to it (pp. 60-71), so there are quite a few rules that applications can implement to get good default results. In this case, the simple rule on p. 67 that "the upper part takes upward-curving ties, the lower part downward-curving ties" should work fine.

**Reply by Jojo-Schmitz** [reply]
Is _Beyond Bars_ volume 2 of _Behind Bars_?

**Reply by infojunkie** [reply]
Page 67 of the 2011 edition, ISBN 0571514561
![p67](https://user-images.githubusercontent.com/92977/158306611-57d6cbce-2b14-4925-8aaa-15ac7c1fa1c1.png)
